cdr_data_dictionary/schema.yaml

```diff
@@ -56,11 +60,11 @@ available_field_item:
     field_type: any(list(str()), str(), null())
     data_provenance: any(list(str()), str(), null())
     source_ppi_module: any(list(str()), str(), null())
-    transformed_by_registered_tier_privacy_methods: any(list(bool(), null()), bool(), null())
+    transformed_by_registered_tier_privacy_methods: bool()
     transformed_by_privacy_methodology_(controlled_tier): any(bool(), null())
-    additional_notes: any(str(), null())
-    transformation_applied_(registered_tier): any(null(), bool())
-    transformation_applied_(controlled_tier): any(null(), bool())
+    additional_notes: any(list(str()), str(), null())
+    transformation_applied_(registered_tier): any(bool(), null())
+    transformation_applied_(controlled_tier): any(bool(), null())
```

Comment:
Can you explain why you had to change the order of the types?

Reply:
i made some changes to the schema, then made some more, and then came back to this.  :)  i just decided to leave the last change (where null is at the end) because it felt better to declare the field's "optional"-ness at the end of the definition.